src/checkout/shared/FormSelectDropdown.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {FC, useContext} from 'react'
+import {
+  Form,
+  FormElementProps,
+  SelectDropdown,
+  SelectDropdownProps,
+} from '@influxdata/clockface'
+
+// Context
+import {CheckoutContext} from 'src/checkout/context/checkout'
+
+type Props = FormElementProps &
+  Omit<SelectDropdownProps, 'onSelect' | 'selectedOption'>
+
+const FormSelectDropdown: FC<Props> = ({label, required, ...props}) => {
+  const {inputs, handleSetInputs} = useContext(CheckoutContext)
+
+  const handleSelect = (value: string): void => {
+    handleSetInputs(props.id, value)
+  }
+
+  return (
+    <Form.Element
+      htmlFor={props.id}
+      label={label}
+      required={required}
+      // errorMessage={meta.touched && meta.error}
```

Comment:
need to add the error validation here. Not sure how important this is since the values can never be empty?

Reply:
Doesn't seem important since it'll never be empty